client/maisie/resources/models.py

```diff
@@ -44,7 +45,8 @@ def upload(
         with self.config.session as session:
             files = {}
             try:
-                files["file"] = open(filename, "rb")
+                files["file"] = open(filename, "rb").read()
```

Comment:
Does this change not interfere with the way models are passed in the `POST` request?

I will elaborate on this issue tomorrow. We need to configure our testing environment ASAP.